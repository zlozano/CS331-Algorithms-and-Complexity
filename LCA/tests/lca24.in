11 () (7)
10 (13) (5)
13 (14,15) (10)
12 () (8)
15 () (13)
14 (16) (13)
17 () (16)
16 (17) (14)
1 (2,3) ()
3 (6) (1)
2 (4,5) (1)
5 (9,10) (2)
4 (7,8) (2)
7 (11) (4)
6 () (3)
9 () (5)
8 (12) (4)
