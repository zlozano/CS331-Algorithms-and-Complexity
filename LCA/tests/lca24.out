 1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1
 1  2  1  2  2  1  2  2  2  2  2  2  2  2  2  2  2
 1  1  3  1  1  3  1  1  1  1  1  1  1  1  1  1  1
 1  2  1  4  2  1  4  4  2  2  4  4  2  2  2  2  2
 1  2  1  2  5  1  2  2  5  5  2  2  5  5  5  5  5
 1  1  3  1  1  6  1  1  1  1  1  1  1  1  1  1  1
 1  2  1  4  2  1  7  4  2  2  7  4  2  2  2  2  2
 1  2  1  4  2  1  4  8  2  2  4  8  2  2  2  2  2
 1  2  1  2  5  1  2  2  9  5  2  2  5  5  5  5  5
 1  2  1  2  5  1  2  2  5 10  2  2 10 10 10 10 10
 1  2  1  4  2  1  7  4  2  2 11  4  2  2  2  2  2
 1  2  1  4  2  1  4  8  2  2  4 12  2  2  2  2  2
 1  2  1  2  5  1  2  2  5 10  2  2 13 13 13 13 13
 1  2  1  2  5  1  2  2  5 10  2  2 13 14 13 14 14
 1  2  1  2  5  1  2  2  5 10  2  2 13 13 15 13 13
 1  2  1  2  5  1  2  2  5 10  2  2 13 14 13 16 16
 1  2  1  2  5  1  2  2  5 10  2  2 13 14 13 16 17
